src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,83 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+
+  h1 {
+    color: $c-text-primary-white;
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+}
+
+.recommender-grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender__grid__material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  .button-favourite {
+    cursor: pointer;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+
+    .icon-favourite {
+      color: $c-text-primary-white;
+      stroke: $c-text-primary-white;
+      fill: none;
+
+      &--filled {
+        stroke: $c-text-primary-white;
+        color: $c-text-primary-white;
+        fill: $c-text-primary-white;
+      }
+    }
+  }
+
+  .cover-container {
+    justify-content: center;
+    width: 100%;
+  }
+
+  @include breakpoint-m {
+    &:last-of-type {
+      display: none;
+    }
+  }
+}
+
+.recommender__grid__material__text {
```

Comment:
Please rework this class name and the class names below.

It has three elements. You could break it into two blocks - something like `.recommender` and `.recommender-material`. Based on that you can create the `.recommender__meta` or `.recommender-material__meta` element. Note that if you do not break it up in two you run into problems with the material title element as the `title` is also used at the top level.